### PR TITLE
Add international letter rates to pricing page

### PR DESCRIPTION
**Pull request by klssmith**
This also updates a few pages where we listed the minimum and maximum letter price.

This shouldn't be merged until we're ready to go live with international letters.

[Pivotal story](https://www.pivotaltracker.com/story/show/171622825)

### CODE REVIEW

File: app/templates/views/pricing/index.html [hunk 1]
```diff
@@ -193,37 +193,39 @@ <h3 class="heading-small" id="international-numbers">Sending text messages to in
...
+          {% call row_heading() %} {{ sheets }} {% endcall %}
+          {{ text_field(second + ' + VAT') }}
+          {{ text_field(first + ' + VAT') }}
+          {{ text_field(international + ' + VAT') }}
         {% endcall %}
       {% endfor %}
     {% endcall %}
   </div>
   <div class="panel panel-border-wide">
     <p class="govuk-body">These prices were updated on 1 October 2019.</p>
```
Comment (klssmith) [inline app/templates/views/pricing/index.html:228]: What date should we give here @karlchillmaid ? We're adding prices, but not changing existing ones, so not sure if this date should change or not

Comment (karlchillmaid) [reply app/templates/views/pricing/index.html:228]: @klssmith Good question.
I think explaining that some have been added and the rest haven't changed is too much info and possibly distracting.
Suggest we put the date that we publish the international rates – even though UK 1st and 2nd remain the same. I checked this with Pete and he's happy.

File: app/templates/views/pricing/index.html [hunk 2]
```diff
@@ -193,37 +193,39 @@ <h3 class="heading-small" id="international-numbers">Sending text messages to in
...
+        ('2 sheets', '40p', '66p', '92p'),
+        ('3 sheets', '45p', '71p', '£1'),
+        ('4 sheets', '50p', '76p', '£1.08'),
+        ('5 sheets', '55p', '81p', '£1.16'),
       ] %}
         {% call row() %}
-          {% call row_heading() %} {{ sheets }} (double-sided) {% endcall %}
-          {{ text_field(second + 'p + VAT') }}
-          {{ text_field(first + 'p + VAT') }}
+          {% call row_heading() %} {{ sheets }} {% endcall %}
```
[the remark this answers is not in the document]
Comment (klssmith) [reply app/templates/views/pricing/index.html:219]: Yes, it's intentional - notes about that in the story https://www.pivotaltracker.com/n/projects/1443052/stories/171622825

File: app/templates/views/features/letters.html [hunk 3]
```diff
@@ -35,8 +35,12 @@ <h3 class="heading heading-small" id="upload-letters">Upload your own letters</h
   <p class="govuk-body">You can create reusable letter templates in Notify, or upload and send your own letters with the Notify API.</p>
   <p class="govuk-body">Read our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.documentation') }}">documentation</a>.</p>
 
+  <h2 class="heading heading-medium">International letters</h2>
```
[the remark this answers is not in the document]
Comment (idavidmcdonald) [reply app/templates/views/features/letters.html:38]: Ah github is doing something weird. It's not showing me the correct line 26 when I view it in this PR. I can see now that it has been changed in master and in production to say 'Notify can send UK letters by first or second class post.`. Not sure why github is showing me the old thing in this PR.

So I guess that makes my second comment fairly redundant

[the remark this answers is not in the document]
Comment (karlchillmaid) [reply app/templates/views/features/letters.html]: Good catch on the h2/h3 thing @idavidmcdonald – I've fixed that now and added an ID for the H3 so we can link directly to this section if we need to.

**Pull request merged**